util/CMakeLists.txt

```diff
@@ -4,9 +4,7 @@ list(REMOVE_ITEM UTIL_SRCS ${PROJECT_SOURCE_DIR}/util/execution/tpl.cpp)
 list(REMOVE_ITEM UTIL_SRCS ${PROJECT_SOURCE_DIR}/util/execution/gen_opt_bc.cpp)
 list(REMOVE_ITEM UTIL_SRCS ${PROJECT_SOURCE_DIR}/util/execution/bytecode_handlers_ir.cpp)
 add_library(util_static STATIC ${UTIL_SRCS})
```

Comment:
Do we want a shared version of `util` as well or punt for now? Curious how big the test binaries end up with that still being static.

Reply:
Seems like tests are tiny even with util statically linked: 
```
~/w/t/b/relwithdebinfo on TerrierShared* du -h hash_index*                                                   
11M     hash_index_test
704K    hash_index_test_stripped
```

Reply:
I'm going to leave it alone. I don't think we want/need a shared version of util. I'll mark this as resolved, but feel free to reopen it and/or comment if you think otherwise.